.github/workflows/push_pr.yml

```
@@ -0,0 +1,109 @@
+name: Push/PR pipeline
+
+on:
+  push:
+    branches:
+      - '**'
+    tags-ignore:
+      - '**'
```

We are trying to converge to a single push_pr workflow, for now we have this:
```suggestion
on:
  push:
    branches:
      - main
  pull_request:
```

---

For now we run this pipeline for any push to any branch, but we exclude run for tags. 
With you change, will we be able to rerun pipeline on any push, or should we create PR to have pipeline started? Will it run new pipeline on each push in open PR?

I am happy to improve those rules, but verify that it will be what we (CAOS/CoreInt) wants from pipeline.

---

In the pipelines we have migrated before (e.g. [nri-kubernetes](https://github.com/newrelic/nri-kubernetes/)) we chose to run the pipelines only on pull requests, rather than on push to any branch. I think the main motivation was to avoid wasting minutes in WIP-ish branches before the PR is opened.

It does re-run the pipeline when new commits are pushed to a PR.

I'm actually fine with either approach, but I'd like for all repos we manage to have the same workflow to avoid later confusion.

---

Sure, if it runs pipeline per each push within open PR, it should be good.